AsyncDisplayKit/ASCollectionView.mm

```diff
@@ -123,7 +124,15 @@ @interface ASCollectionNode ()
 - (instancetype)_initWithCollectionView:(ASCollectionView *)collectionView;
 @end
 
+static BOOL _enableGlobalDebugCounts = NO;
+
 @implementation ASCollectionView
+{
+  ASTextNode *_debugCountsTextNode;
+  NSUInteger _globalDebugNodeCount;
```

Comment:
global is not the right term for these -- though the __shouldDisplayDebugCounts variable is truly global, these are per-instance so we should probably just name them _debugNodeCount, _debugViewCount, etc

Let's actually try to eliminate these if possible...